netket/optimizer/sr/sr_treemv_logic.py

```diff
@@ -20,7 +20,7 @@
 from functools import partial
 
 from netket.stats import sum_inplace, subtract_mean
-from netket.utils import n_nodes
+from netket.utils.mpi import n_nodes
```

Comment:
This needs to be updated in `netket.tools` as well, right?

Reply:
yes, I fixed it.

Reply:
we should add CI to the tools as well, at some tpoint